srcalc/src/main/webapp/WEB-INF/views/admin/summaryReportForm.jsp

```diff
@@ -0,0 +1,71 @@
+<%@ taglib uri="/WEB-INF/srcalc.tld" prefix="srcalc" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib prefix="spring" uri="http://www.springframework.org/tags" %>
+<%@ taglib uri="http://www.springframework.org/tags/form" prefix="form" %>
+
+<srcalc:adminPage title="Summary Report Parameters">
+
+<section>
+    <h2>Summary Report Parameters</h2>
+    <form:form id="summaryReportParametersForm" cssClass="srcalcForm attributeEditForm"
+        method="post" commandName="reportParameters">
+    <%-- Display any object-wide errors --%><form:errors cssClass="error"/>
+    <%-- See srcalc.css for why HTML tables. --%>
+    <table>
+    <tbody>
+        <tr>
+        <td class="attributeName">Date Range</td>
+        <td>
+            <form:input path="minDate" cssClass="dateEntry" size="10" />
+            <form:errors path="minDate" cssClass="error" />
+            through
+            <form:input path="maxDate" cssClass="dateEntry" size="10" />
+            <form:errors path="maxDate" cssClass="error" />
+        </td>
+        </tr>
+        <tr>
+        <td class="attributeName">Specialties</td>
```

Comment:
The specialties are all just auto-wrapped right now and I think it would look better for them to be in line vertically with each other or organized in a better way. We can discuss.